### PR TITLE
Sign in/up error handling and patient navbar

### PR DESCRIPTION
**Pull request by jakepeyser**

### CODE REVIEW

File: app/components/navbar/NavbarTab.js [hunk 1]
```diff
@@ -6,12 +6,12 @@ import { toolbar, textLight } from '../colors'
 // Regular text input
 export default props => (
   <div className="navbar-item">
-    <Link to="/">
+    <Link to={`/${props.type}`}>
       <StepsFlatButton
-        label={ props.label }
+        label={ props.type.charAt(0).toUpperCase() + props.type.slice(1) }
         textColor={ textLight }
```
Comment (Hitenp1) [inline app/components/navbar/NavbarTab.js]: What is the props.type?

Comment (jakepeyser) [inline app/components/navbar/NavbarTab.js]: The type of tab (i.e. patients, messages, etc)

File: app/utils.js [hunk 2]
```diff
@@ -9,3 +11,10 @@ export const isBrowser = () => {
 
 // Return formatted full name of user
 export const fullName = user => `${user.first_name} ${user.last_name}`;
+
+// Redirect page for logged in user based on role
+export const loginRedirect = role => {
+  if (role === THERAPIST) return '/dashboard';
+  else if (role === PATIENT) return '/plan';
+  else return '/';
```
Comment (Hitenp1) [inline app/utils.js]: does the therapist have a dashboard?  I thought their redirect page on the mvp was the patient list (with the add patient button)

Comment (jakepeyser) [inline app/utils.js]: good point. will change this to `/patients` and update navbar accordingly

**Pull request merged**